default.nix

```diff
@@ -6,72 +6,109 @@ with pkgs;
 
 let
   root = ./.;
-  mainnet = {
+  protocol005 = rec {
```

Comment:
```suggestion
  protocol005 = {
```

Reply:
Nope, this one is required :upside_down_face: